runtime/runtime-core/common/test/aws/smithy/kotlin/runtime/retries/impl/StandardRetryPolicyTest.kt

```diff
@@ -47,9 +47,9 @@ class StandardRetryPolicyTest {
     }
 
     @Test
-    fun testClientException() {
+    fun testNonRetryableClientException() {
         val result = test(ClientException())
-        assertEquals(RetryDirective.RetryError(RetryErrorType.ClientSide), result)
+        assertEquals(RetryDirective.TerminateAndFail, result)
     }
```

Comment:
**Nit**: Missing a new unit test for `ClientException` that is retryable.

Reply:
Yes I see that now, I thought thats what the other function you commented on was doing but I see now it's not.